examples/charm++/cuda/gpudirect/jacobi2d/scripts/lassen/ncu.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Out of curiosity, why are we shipping these lassen/summit batch scripts?

Reply:
Mainly just because I used these scripts for benchmarking and didn't want them to get lost. Also I thought it might be useful for people that wanted to run on those machines. Do you think it's better if these are left out?

Reply:
I'm not sure, let's discuss in core?

Reply:
👍

Reply:
Have we resolved this?

Reply:
@matthiasdiener Thanks for the reminder will bring this up soon

Reply:
Will remove the scripts from this PR and move them to a private repo/branch